src/alignment/model_utils.py

```diff
@@ -69,10 +69,14 @@ def get_tokenizer(model_args: ModelArguments, data_args: DataArguments) -> PreTr
         if model_args.tokenizer_name_or_path is None
         else model_args.tokenizer_name_or_path,
         revision=model_args.model_revision,
+        trust_remote_code=model_args.trust_remote_code
     )
     if tokenizer.pad_token_id is None:
         tokenizer.pad_token_id = tokenizer.eos_token_id
 
+    if tokenizer.bos_token_id is None:
+        tokenizer.bos_token_id = tokenizer.eos_token_id
```

Comment:
bos_token need to set by tokenizer (a tokenizer inhert from PreTrainedTokenizer need take care of this propoerty. We can asset and raise error if find it missing.